packages/pieces/google-gemini/src/lib/actions/generate-content-from-image.action.ts

```diff
@@ -0,0 +1,56 @@
+import { HttpMethod, httpClient } from '@activepieces/pieces-common';
+import { googleGeminiAuth } from '../../index';
+import { Property, createAction } from "@activepieces/pieces-framework";
+
+
+export const generateContentFromImageAction = createAction({
+    description: 'Generate content using Google Gemini using the "gemini-pro-vision" model',
+    displayName: 'Generate Content from Image',
+    name: 'generate_content_from_image',
+    auth: googleGeminiAuth,
+    props: {
+        prompt: Property.LongText({
+            displayName: 'Prompt',
+            required: true,
+            description: 'The prompt to generate content from.'
+        }),
+        image_base64: Property.LongText({
```

Comment:
You can replace image_base64 and mime_type with Property.File 

This accept url / base64 and convert it to developer friendly format.

Reply:
It's done @abuaboud